documentation/README.md

```diff
@@ -6,7 +6,7 @@ Here you find OSHDB related documentation material:
   This tutorial teaches how to use the OSHDB-API in Java.
 * [OSHDB Manual](manual/README.md)<br>
   Explains the design of the OSHDB data model and shows the different features of the OSHDB API and how they can be used to efficiently query the OSM history data.
-* [Examples](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples)<br>
+* [Examples](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples) <br>
```

Comment:
does this really change anything? (also below)

Reply:
Apparently in the GitHub Markdown parser it doesn't. In other markdown parser it ensures that the link is rendered properly. Should I remove the space again?

Reply:
I was just wondering why you didn't also change it in line 7 :thinking:

Reply:
True. I didn't change it, because IDEAs markdown parser parsed them correctly. Made it consistent: https://github.com/GIScience/oshdb/compare/0064f8267194fe69fa73afd886725215077d7d4e..4a469449e9285cbb88a6786d9479d202b25e334f